### PR TITLE
Desktop Version

### PR DESCRIPTION
**Pull request by gaks1**
What's done:
 Made a desktop version of the mobile portfolio website by adding a media query of 769px.
1. Changed font sizes on all texts.
2.  Changed image position in the works section.
3. Changed the about section placement.
4. Changed the background images.
5. Updated readme.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,6 +1,1178 @@
 
 @import url('https://fonts.googleapis.com/css?family=Poppins');
 
+@media screen and (min-width: 769px) {
+  .ul {
```
Comment (joskalenda) [inline style.css]: - [x] Well done on respecting the requirement, but I think your media query starts from `769px`and the output on the browser is not responsive at all, please take a look at this record and make the necessary changes to improve your styling



- [ ] Please make sure your hero text uses the same font weight property as per the original design

- [ ] Your social media icons are not well styled, please compare the 2 images and make the necessary changes

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @gaks1 , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |

## Highlights

- [ ].. ✅ Good design
- [ ].. ✅ Green linters
- [ ].. ✅ Good readme file

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -1,6 +1,1175 @@
...
+    line-height: 60px;
+    display: flex;
+    align-items: center;
+    color: #172b4d;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+    letter-spacing: 0.37px;
+    width: 500px;
+  }
```
Comment (NduatiKagiri) [inline style.css]: - [ ] You are still not using the right values as specified in the design. Please make sure you apply the right font size and line heights on this text.

  Isaac1

-   Please also note that the heading should break after your name and 'Glad' should start on the following line. ou can implement this by using line breaks like this : `<hx>I’m Isaac <br>Glad to see you!</hx>`

**Review by NduatiKagiri — CHANGES_REQUESTED** [review]
Hi @gaks1 👋 ,

# CHANGES REQUIRED ♻️  

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

so close


 ## Highlights ✔️ 
✅ No Linters errors
✅ GitHub Flow observed
✅ Good PR description
✅ Good/smooth UI


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

You can also consider:
-   Please rename your markup from `main.html` to `index.html`. A good reason for this is that web servers always search for `index.html` in the parent directory when the project is executed, and if the file is not found, a `404` error is returned by the server which means that the requested project files cannot be found.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AbayomiOlaoye — APPROVED** [review]
Hi @gaks1 :clap: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

applause

## Highlights
- Great implementation of reviewer's suggestions :heavy_check_mark: 
- Good UI implementation :heavy_check_mark: 
- Linter checks are passing :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**